app/Providers/AppServiceProvider.php

```diff
@@ -17,6 +18,9 @@ public function boot()
       View::composer(
         'partials.components.country-select','App\Http\ViewComposers\CountrySelectViewComposer'
       );
+
+      Schema::defaultStringLength(191);
```

Comment:
Why is that? There are some places in the code where we wanted to limit the number of characters in an input field to 255 characters. If we put this, does that mean that the default VARCHAR is limited to 191?

Reply:
Just looking through some PRs and noticed this one.

What would happen to the installs of Monica that have a varchar-field that have values of 192 characters or longer? Truncated data is not so fun.

If breaking changes is fine, just forcing users to enable `innodb_large_prefix` on the older MySQL-installs would be a much more elegant solution and wouldn't cause any data loss.

https://stackoverflow.com/questions/43153719/laravel-5-4-migrate-key-is-too-long-error/43162269#43162269